plugins/inventory/proxmox.py

```diff
@@ -362,6 +362,36 @@ def _get_node_ip(self, node):
             except Exception:
                 return None
 
+    def _get_lxc_interfaces(self, properties, node, vmid):
+        try:
+            ret = self._get_json("%s/api2/json/nodes/%s/lxc/%s/interfaces" % (self.proxmox_url, node, vmid))
+        except Exception:
+            return
```

Comment:
Simply ignoring all errors seems wrong to me. For which reasons can this call fail?

Reply:
I did this to avoid introducing a breaking change, since this particular API is somewhat new (introduced in proxmox 8.1). 

Maybe a version check makes more sense?

Reply:
@felixfontein I've made some small changes.

1. Requests for a container that are not running will return null, so there is an explicit check for the status fact.
2. From my tests and a bit of reading, previous versions of the API will return 501 for requests to unknown endpoint. I verified this behavior with a server running version 7.

Adding an explicit version check can be done, but is more complicated. How many versions back does this plugin need to support?

Reply:
> Adding an explicit version check can be done, but is more complicated.

Checking for 501 sounds good to me.

> How many versions back does this plugin need to support?

I cannot find anything in the documenation, so basically "all versions the plugin worked with so far".